src/foundry/foundry.js/hooks.d.ts

```diff
@@ -296,28 +303,26 @@ declare global {
       ) => boolean | void;
 
       /**
-       * @remarks This is called after the initial {@link SceneControls} have been set up.
-       * @param controls - the created controls
+       * A hook event that fires when the Scene controls are initialized.
+       * @param controls - The SceneControl configurations
        * @remarks This is called by {@link Hooks.callAll}.
        * @see {@link SceneControls#_getControlButtons}
        */
       getSceneControlButtons: (controls: SceneControl[]) => unknown;
 
       /**
-       * @remarks This is called after getting the {@link ContextMenu} options for the {@link SceneNavigation}, but before
-       * creating the ContextMenu.
-       * @param jq             - the JQuery of the ContextMenu parent element
-       * @param contextOptions - the already created ContextMenuEntrys
+       * A hook event that fires when the context menu for a SceneNavigation entry is constructed.
+       * @param html         - The HTML element to which the context options are attached
+       * @param entryOptions - The context menu entries
        * @remarks This is called by {@link Hooks.call}.
        * @see {@link SceneNavigation#activateListeners}
        */
       getSceneNavigationContext: (jq: JQuery, contextOptions: ContextMenuEntry[]) => boolean | void;
 
       /**
-       * @remarks This is called after getting the {@link ContextMenu} options for a {@link PlayerList} user, but before
-       * creating the ContextMenu.
-       * @param jq             - the JQuery of the ContextMenu parent element
-       * @param contextOptions - the already created ContextMenuEntrys
+       * A hook event that fires when the context menu for a PlayersList entry is constructed.
```

Comment:
The parameters in the function definition should be renamed according to the documentation.